README.md

```diff
@@ -197,6 +201,75 @@ NDArray succesfully created in btune_config.b2nd
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0389 |      8.01x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0371 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0361 |      8.01x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0377 |      8.01x |    CODEC_FILTER |    HARD | -
+```
+The tradeoff used tells Btune that you care a lot about compression ratio but not as much
+about speed or the quality loss. According to that, Btune uses the `blosc2_grok` codec 
+and manages to achieve a compression ratio of 8x in exchange for losing some quality.
+
+Like in the traditional Btune, you can use the `BTUNE_TRADEOFF` environmnet
+variable to change the tradeoff:
+```shell
+BTUNE_TRADEOFF="(0.5, 0, 0.5)" BTUNE_TRACE=1 python lossy.py MI04_020751.tif
+
+-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+Btune version: 1.1.2
+Performance Mode: COMP, Compression tradeoff: (0.500000, 0.000000, 0.500000), Bandwidth: 20 GB/s
+Behaviour: Waits - 0, Softs - 0, Hards - 0, Repeat Mode - (null)
+Behaviour: Waits - 0, Softs - 5, Hards - 10, Repeat Mode - STOP
+INFO: Created TensorFlow Lite XNNPACK delegate for CPU.
+TRACE: time load model: 0.000283
+|    Codec   | Filter | Split | C.Level | C.Threads | D.Threads |  S.Score  |  C.Ratio   |   Btune State   | Readapt | Winner
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0134 |         4x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0352 |         4x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0374 |         4x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0366 |         4x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0393 |         4x |    CODEC_FILTER |    HARD | W
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0365 |         4x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0357 |         4x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0368 |         4x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0365 |         4x |    CODEC_FILTER |    HARD | -
+|       grok |      0 |     0 |       5 |         4 |         4 |    0.0372 |         4x |    CODEC_FILTER |    HARD | -
+```
+In this case, because the quality is more important than before, Btune uses
+the `blosc2_grok` codec with compression ratio 4x, which losses less info than
+the 8x.
```

Comment:
Is it possible to show an example of prediction of a lossy filter + lossless codec?

Reply:
Fixed in [dc81f86](https://github.com/Blosc/Blosc2-Btune/pull/13/commits/dc81f869229f601c1c8f24d91e12be129c373486).